great_expectations/expectations/metrics/column_map_metrics/column_values_match_regex.py

```diff
@@ -36,3 +52,51 @@ def _sqlalchemy(cls, column, regex, _dialect, **kwargs):
     @column_condition_partial(engine=SparkDFExecutionEngine)
     def _spark(cls, column, regex, **kwargs):
         return column.rlike(regex)
+
+
+class ColumnValuesMatchRegexCount(MetricProvider):
+    metric_name = "column_values.match_regex.count"
+
+    metric_value_kwargs = ("regex",)
+
+    @metric_value(engine=PandasExecutionEngine)
+    def _pandas(*, metrics, **kwargs):
+        return metrics[
+            f"column_values.not_match_regex.{SummarizationMetricNameSuffixes.UNEXPECTED_COUNT.value}"
+        ]
```

Comment:
Pattern matching is the only reason I remembered to do this this way. The `unexpected_count` suffix is something you can add to any "condition partial" metric. The reason it's `unexpected_count` (as opposed to `count`, which I have erroneously said could be used recently), is because Expectation Validation Results always return information about the "bad" data (e.g. `UnexpectedRowsExpectation`).